### ISSUE TITLE
[Feature]: I would like to use ray with nodes without GPU

### ISSUE THREAD

**Issue by bisontim** [vllm-project/vllm#11131]
### 🚀 The feature, motivation and pitch

hi,
I would like to use the fantastic https://docs.vllm.ai/en/stable/serving/distributed_serving.html feature.
i understood it works with GPU only.
I tried to modify the run_cluster.sh removing the --gpus all 
but it doesn't work.
Could you give me details of this requested feature?
Thanks a lot
Mario

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by ruisearch42** [comment]
Hi @bisontim , which accelerators would you like to use?

**Comment by bisontim** [comment]
Sorry... I am not an expert..What do you mean accelerator?
I have hosts without gpu and I tried to setup with ray.

**Comment by ruisearch42** [comment]
Yeah, I meant what types of accelerators were you using, TPU? NPU? HPU?
Can you also describe your use-case in more detail?
